src/main.ts

```diff
@@ -0,0 +1,52 @@
+import { registerCommands } from '#lib/utilities/register-commands';
+import { envParseInteger, envParseString, setup } from '@skyra/env-utilities';
+import { Client, container } from '@skyra/http-framework';
+import { init, load } from '@skyra/http-framework-i18n';
+import { setInvite, setRepository } from '@skyra/shared-http-pieces';
+import '@skyra/shared-http-pieces/register';
```

Comment:
This doesn't have to be at the top specifically?

Reply:
No, but I can make it a dynamic import.